ariadne/src/Ariadne/Config/CLI.hs

```diff
@@ -513,9 +519,9 @@ cliConfigurationOptionsParser = do
     ]
   return CLI_ConfigurationOptions{..}
 
-cliCommonArgsParser :: Opt.Parser CLI_CommonArgs
+cliCommonArgsParser :: Opt.Parser (CLI_CommonArgs conf)
 cliCommonArgsParser = do
-  cli_logConfig <- optional $ strOption $ mconcat
+  cli_logConfig <- (<$>) File . optional . strOption . mconcat $
```

Comment:
It's a bit strange to see such usage of `(<$>)`, because there is `fmap` (same for any other operator which has a well known alias function), but ok.